2021-07-01 20:03:50,702 INFO    MainThread:5352 [wandb_setup.py:_flush():69] setting env: {}
2021-07-01 20:03:50,702 INFO    MainThread:5352 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 20:03:50,702 INFO    MainThread:5352 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 20:03:50,702 INFO    MainThread:5352 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_200350-3qk4j3ks\logs\debug.log
2021-07-01 20:03:50,702 INFO    MainThread:5352 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_200350-3qk4j3ks\logs\debug-internal.log
2021-07-01 20:03:50,749 INFO    MainThread:5352 [wandb_init.py:_jupyter_setup():288] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000204C2A47B38>
2021-07-01 20:03:50,749 INFO    MainThread:5352 [wandb_init.py:init():370] calling init triggers
2021-07-01 20:03:50,749 INFO    MainThread:5352 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-01 20:03:50,749 INFO    MainThread:5352 [wandb_init.py:init():392] re-initializing run, found existing run on stack: 2p8l05ts
2021-07-01 20:03:50,749 INFO    MainThread:5352 [wandb_run.py:finish():1294] finishing run abhishekdiphu/IBM-Project/2p8l05ts
2021-07-01 20:03:50,749 INFO    MainThread:5352 [wandb_run.py:_atexit_cleanup():1594] got exitcode: 0
2021-07-01 20:03:50,764 INFO    MainThread:5352 [wandb_run.py:_restore():1566] restore
2021-07-01 20:03:51,715 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 104158
  total_bytes: 104158
}

2021-07-01 20:03:52,105 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 1
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 104158
  total_bytes: 104158
}

2021-07-01 20:03:52,763 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 104158
  total_bytes: 108228
}

2021-07-01 20:03:53,107 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 104158
  total_bytes: 108228
}

2021-07-01 20:03:53,217 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 107184
  total_bytes: 108228
}

2021-07-01 20:03:53,326 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 108228
  total_bytes: 108228
}

2021-07-01 20:03:53,427 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 108228
  total_bytes: 108228
}

2021-07-01 20:03:53,537 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 108228
  total_bytes: 108228
}

2021-07-01 20:03:53,646 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 108228
  total_bytes: 108228
}

2021-07-01 20:03:53,756 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 108228
  total_bytes: 108228
}

2021-07-01 20:03:53,865 INFO    MainThread:5352 [wandb_run.py:_wait_for_finish():1716] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  media_count: 3
  artifact_count: 3
}
pusher_stats {
  uploaded_bytes: 108228
  total_bytes: 108228
}

2021-07-01 20:03:54,968 INFO    MainThread:5352 [wandb_run.py:_show_summary():1871] rendering summary
2021-07-01 20:03:54,983 INFO    MainThread:5352 [wandb_run.py:_show_history():1909] rendering history
2021-07-01 20:03:54,983 INFO    MainThread:5352 [wandb_run.py:_show_files():1938] logging synced files
2021-07-01 20:03:54,983 INFO    MainThread:5352 [wandb_init.py:init():419] starting backend
2021-07-01 20:03:54,983 INFO    MainThread:5352 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=spawn, using: spawn
2021-07-01 20:03:54,999 INFO    MainThread:5352 [backend.py:ensure_launched():135] starting backend process...
2021-07-01 20:03:55,124 INFO    MainThread:5352 [backend.py:ensure_launched():140] started backend process with pid: 2248
2021-07-01 20:03:55,124 INFO    MainThread:5352 [wandb_init.py:init():424] backend started and connected
2021-07-01 20:03:55,140 INFO    MainThread:5352 [wandb_run.py:_label_probe_notebook():845] probe notebook
2021-07-01 20:03:55,140 INFO    MainThread:5352 [wandb_run.py:_label_probe_notebook():852] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-07-01 20:03:55,140 INFO    MainThread:5352 [wandb_init.py:init():472] updated telemetry
2021-07-01 20:03:55,140 INFO    MainThread:5352 [wandb_init.py:init():491] communicating current version
2021-07-01 20:03:57,315 INFO    MainThread:5352 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.10.33 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-07-01 20:03:57,315 INFO    MainThread:5352 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-07-01 20:03:57,746 INFO    MainThread:5352 [wandb_init.py:init():529] starting run threads in backend
2021-07-01 20:03:58,137 INFO    MainThread:5352 [wandb_run.py:_console_start():1624] atexit reg
2021-07-01 20:03:58,137 INFO    MainThread:5352 [wandb_run.py:_redirect():1498] redirect: SettingsConsole.WRAP
2021-07-01 20:03:58,137 INFO    MainThread:5352 [wandb_run.py:_redirect():1535] Wrapping output streams.
2021-07-01 20:03:58,137 INFO    MainThread:5352 [wandb_run.py:_redirect():1559] Redirects installed.
2021-07-01 20:03:58,137 INFO    MainThread:5352 [wandb_init.py:init():554] run started, returning control to user process
2021-07-01 20:03:59,074 INFO    MainThread:5352 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'outlier_candidates') {'id': 'wandb/outliers/v1', 'historyFieldSettings': {'key': 'outlier_candidates', 'x-axis': '_step'}} None
2021-07-01 20:04:15,000 INFO    MainThread:5352 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'learning_curve') {'id': 'wandb/learning_curve/v1', 'historyFieldSettings': {'key': 'learning_curve', 'x-axis': '_step'}} None
2021-07-01 20:04:19,574 INFO    MainThread:5352 [wandb_run.py:_config_callback():873] config_cb ('_wandb', 'viz', 'residuals') {'id': 'wandb/residuals_plot/v1', 'historyFieldSettings': {'key': 'residuals', 'x-axis': '_step'}} None
2021-07-01 20:04:27,951 INFO    MainThread:5352 [wandb_setup.py:_flush():69] setting env: {}
2021-07-01 20:04:27,951 INFO    MainThread:5352 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 20:04:27,951 INFO    MainThread:5352 [wandb_setup.py:_flush():69] setting login settings: {}
2021-07-01 20:04:27,951 INFO    MainThread:5352 [wandb_init.py:_log_setup():337] Logging user logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_200427-26z9mdal\logs\debug.log
2021-07-01 20:04:27,951 INFO    MainThread:5352 [wandb_init.py:_log_setup():338] Logging internal logs to C:\Users\abhishek buragohaibn\Desktop\COURSERA\IBM_AI_Enterprise_Workflow\AI_capstone\solution-guidance\wandb\run-20210701_200427-26z9mdal\logs\debug-internal.log
2021-07-01 20:04:27,951 INFO    MainThread:5352 [wandb_init.py:init():370] calling init triggers
2021-07-01 20:04:27,951 INFO    MainThread:5352 [wandb_init.py:init():377] wandb.init called with sweep_config: {}
config: {}
2021-07-01 20:04:27,951 INFO    MainThread:5352 [wandb_init.py:init():392] re-initializing run, found existing run on stack: 3qk4j3ks
2021-07-01 20:04:27,951 INFO    MainThread:5352 [wandb_run.py:finish():1294] finishing run abhishekdiphu/IBM-Project/3qk4j3ks
